variables.tf

```diff
@@ -362,3 +359,8 @@ variable "permissions_boundary" {
   description = "ARN of the policy that will be used to set the permissions boundary for all IAM Roles created by this module"
   default     = null
 }
+
+variable "name_prefix" {
+  type        = string
+  description = "A string to prepend to names of the resources in the cluster"
+}
```

Comment:
I don't think this variable is being used by the root module, but maybe I'm missing something.

Reply:
done